cmd/namespace_registry_serve.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	// "github.com/pelicanplatform/pelican/config"
+	"github.com/pelicanplatform/pelican/namespace-registry"
+	"github.com/pelicanplatform/pelican/web_ui"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	// "github.com/spf13/viper"
+)
+
+
+func serveNamespaceRegistry( /*cmd*/ *cobra.Command /*args*/, []string) error {
+	log.Info("Initializing the namespace registry's database...")
+
+	// Initialize the registry's sqlite database
+	err := nsregistry.InitializeDB()
+	if err != nil {
+		return fmt.Errorf("Unable to initialize the namespace registry database: %q", err)
```

Comment:
Switch to `errors.Wrapf` to wrap the error instead of converting the underlying error to a string.